a2a4j-core/src/main/java/io/github/a2ap/core/server/impl/InMemoryTaskStore.java

```diff
@@ -37,6 +41,14 @@ public void save(Task task) {
 
     @Override
     public Task load(String taskId) {
+
```

Comment:
`
```shell 
$ curl -X POST http://localhost:8089/a2a/server   -H "Content-Type: application/json"   -d '{
    "jsonrpc": "2.0",
    "method": "tasks/get",
    "params": {
      "taskId": "task-abc123"
    },
    "id": "get-task-1"
  }'
{"jsonrpc":"2.0","error":{"code":-32603,"message":"Internal error","data":"Cannot invoke \"Object.hashCode()\" because \"key\" is null"},"id":"get-task-1"}
```